src/components/issues/issueContainer.js

```diff
@@ -19,7 +19,12 @@ class IssueContainer extends React.Component {
     this.setState(current => {
       return { ...current, loading: true,offset: 0,showingIssues:[] };
     });
-    let issues = await getIssues(this.props.project);
+    let issues;
+    try {
+    issues=  await getIssues(this.props.project)
+    }catch (err){
+      console.log(err)
```

Comment:
set a state say `this.setState({ error: false })` and then inside the render add the markup to say "Couldn't fetch the issues. Please try again.".

That way when something errors, user will get to know that something breaked otherwise they will be clueless.

Reply:
yes. I should do that.

Reply:
should I just this exact method or the error coming for backend?

Reply:
it's @siddhant1 call